_posts/2025-06-17-seeds-to-system-funding.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Announcing Funding for the Seeds to Systems Initiative"
+authors:
+- malvikasharan
+- yochannah
+- pherterich
+image: https://unsplash.com/photos/diagram-RIvSJTiGwLc
```

Comment:
```suggestion
image: https://images.unsplash.com/photo-1664526936810-ec0856d31b92
```
This is the link to the actual image you want - the previous one goes to a page containing the image, **and** others. :-)